Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -52,6 +52,135 @@ function Module:reserveIntValue(max_value)
 	self:allocateInt(max_value)
 end
 
+local function cap(value, limits, cycle)
+	if cycle then
+		if value < limits[1] then
+			return limits[2]
+		end
+		if value > limits[2] then
+			return limits[1]
+		end
+	else
+		if value <= limits[1] then
+			return limits[1]
+		end
+		if value >= limits[2] then
+			return limits[2]
+		end
+	end
+	return value
+end
+
+--- Uses SetCommand and set_argument_value instead of performClickableAction()
+--- @param identifier string the unique identifier for the control
+--- @param device_id integer the dcs device id
+--- @param command integer the dcs command
+--- @param arg_number integer the dcs argument number
+--- @param step number the amount to increase or decrease dcs data by with each step
+--- @param limits number[] a length-2 array with the lower and upper bounds of the data as used in dcs
+--- @param output_map string[]? an array of string values to output for inputs across the range of values, or nil if none
+--- @param cycle boolean | "skiplast" true if infinite rotary, false if limited rotary, skiplast functionality unclear
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+function Module:defineSetCommandTumb(identifier, device_id, command, arg_number, step, limits, output_map, cycle, category, description)
```

Comment:
Will you be adding unit tests for this command?

Reply:
Added now, copied setTumb